PlexServiceCommon/LogWriter.cs

```diff
@@ -42,16 +44,13 @@ public static class LogWriter {
 
         public static async Task<string> Read() {
             var log = string.Empty;
-            if (!File.Exists(LogFile)) {
-                Log.Debug("No logfile found??");
-                return log;
-            }
-
+            var target = GetLatestLog();
+            if (target == string.Empty) return log;
```

Comment:
Please change to use string.IsNullOrEmpty(target) instead of a value test against empty string.

Reply:
Updated